modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -67,6 +67,46 @@ function oe_content_publication_post_update_00003() {
     $reference_field['_core']['default_config_hash'] = Crypt::hashBase64(serialize($reference_field));
     $field_config->create($reference_field)->save();
   }
+}
+
+/**
+ * Create the new fields in the Publication content type.
+ */
+function oe_content_publication_post_update_00004(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_publication') . '/config/post_updates/00004_create_fields');
```

Comment:
We can create oe_publication_contacts with another fields to 00004 (need to add default_config_hash to the yml files then).

Reply:
I added the `oe_publication_contacts` fields to the other new fields. I added the config_hash values also.

Reply:
So you can remove creation of oe_publication_contacts field in 00003